### ISSUE TITLE
Cryo powerloss breakout leads to animation permalock

### ISSUE THREAD

**Issue by SCOOPS-D-Ore** [tgstation/tgstation#70469]
## Reproduction:

I've had this happen only once, but here are the events that lead up to it:

1. Go into O2 crit due to forgetting internals.
2. As a lizard, get put into Cryo.
3. Healed out of crit, but not yet enough to get auto-ejected.
4. have the APU for the cryo turn off before heal completes.
5. Be broken out of cryo by your caregiver, flooding the room with N2 and cold air.
(Note, it would seem the anomalous effect of whatever bug this is stats here, as a SLIP is noted happening as soon as the cryochamber tube breaks. The sound for a slip plays, however the playermodel does not go horizontal.
At this point, you should be stuck bolt-upright regardless of what happens to you.

## In summary:

 The overall effect of this bug is the player becomes immobile, but can speak and turn in place using Control+Directional keys. The player cannot use their hands to interact with bags or items, but can use UI buttons to turn on/off internals. 

The perma-standing effect still continues no matter what happens to the player model. This includes weapons fire, unconsciousness after the bug has started, being moved or tabled, pushed, N2 slept, and even death. 

Final note: The resolution in this case was a helpful admin reviving a second body from the ghosted player. (Thanks, Trexter555!)
